deployment/quick-start/services-configuration.yaml.template

```diff
@@ -128,3 +128,7 @@ rest-server:
 # uncomment following section if you want to customize the port of pylon
 # pylon:
 #  port: 80
+
+# uncomment following section if you want to customize the docker path of cleaner
```

Comment:
>uncomment [](start = 2, length = 9)

remove these three lines if you have not implemented it yet.

Reply:
Already implemented